networkx/algorithms/connectivity/tests/test_disjoint_paths.py

```diff
@@ -0,0 +1,219 @@
+from nose.tools import assert_equal, assert_true, assert_false, raises
+
+import networkx as nx
+from networkx.algorithms.flow import edmonds_karp
+from networkx.algorithms.flow import preflow_push
+from networkx.algorithms.flow import shortest_augmenting_path
+from networkx.utils import pairwise
+
+flow_funcs = [edmonds_karp, preflow_push, shortest_augmenting_path]
+
+msg = "Assertion failed in function: {0}"
+
+
+def is_path(G, path):
```

Comment:
Nice. I like that implementation better. Now I also notice that forgot to check that edge disjoint paths are actually paths in the tests. Regarding #1921 note that edge disjoint paths are not necessarily simple (node disjoint paths are).